README.md

```diff
@@ -216,14 +284,28 @@ public int ExampleInteger
     }
 }
 ```
+#### Hungarian notation
+
+We don't do that here. It's crucial to note that Hungarian notation is considered a suboptimal practice in coding standards.
 
-Arrays follow the same naming rules as above, but should be named as a plural noun.
+```cs
+// good
+private int targetAmount;
+private Player player;
+private ExampleStruct currentStruct;
+
+//bad
+private int _intTargetAmount;
+private Agent _agentAgent;
+private ExampleStruct _exampleStructCurrentStruct;
+```
+This also apply to collections. They follow the same naming rules as mentioned before, but should be named as a plural noun.
```

Comment:
wtf is this?

Reply:
This is a showcase of Hungarian notation. Here you can see why we are not going to use it.

Reply:
I will change some names to be more clear.